### ISSUE TITLE
Judge is a somewhat common surname

### ISSUE THREAD

**Issue by GoogleCodeExporter** [gdsaxton/python-nameparser#3]
```
What steps will reproduce the problem?
1. use test name 'Lisa Judge'
2.
3.

What is the expected output? What do you see instead?
Expected name.first == 'Lisa' and name.last = 'Judge'.  Got
name.title = 'Judge' and name.last = 'Lisa' instead.


What version of the product are you using? On what operating system?
0.1.2 on aix with python 2.6

Please provide any additional information below.
I removed judge from the title list for my install and got the output I 
desired.  Was thinking about implementing some sort of "number of pieces" rule 
combined with "location of title" to try to make a better guess.
```

Original issue reported on code.google.com by `slestak...@gmail.com` on 18 Aug 2010 at 7:10

**Comment by GoogleCodeExporter** [comment]
```
you're right, it's probably a more common surname than title. How was the input 
formatted in your case?

 * Lisa Judge
 * Judge, Lisa
```

Original comment by `dere...@gmail.com` on 18 Aug 2010 at 7:19

**Comment by GoogleCodeExporter** [comment]
```
'Lisa Judge'

IMO, we might be able to use the presence if a comma to help id if firstpiece 
is a surname or fname.  In the absence of a comma, if Judge is first, I would 
call it title.  If Judge is not first I would call it surname.  

Maybe another dict of QUESTIONABLE_TITLE, kind of along the lines of how you 
handled 'Hon.' where there is a slight bit of positional testing.  I wouldnt 
subject 1stLt or Dr., however, Doctor spelled out is a surname.
```

Original comment by `slestak...@gmail.com` on 18 Aug 2010 at 7:42

**Comment by GoogleCodeExporter** [comment]
```
I also added 'di' to the PREFICES list.
```

Original comment by `slestak...@gmail.com` on 18 Aug 2010 at 7:43

**Comment by GoogleCodeExporter** [comment]
```
I added 'di'. had some formatting changes that i hadn't committed too, sorry if 
it causes a conflict. Also removed 'judge' from TITLES and added an unused 
QUESTIONABLE_TITLES list.

right now the parser treats these two as equivalent. 

 * Lisa Judge
 * Judge, Lisa

I guess if there's only 2 parts then Judge is pretty likely to be a last name, 
regardless of position or comma. If there's 3 parts it's much more likely it's 
a title..

```

Original comment by `dere...@gmail.com` on 18 Aug 2010 at 8:09

**Comment by GoogleCodeExporter** [comment]
```
Would it be possible to turn on zip or tarball exports of you svn repo?  I dont 
use svn, but I would like to track development and can offer patches for name 
adjustments I make.
```

Original comment by `slestak...@gmail.com` on 18 Aug 2010 at 9:11

**Comment by GoogleCodeExporter** [comment]
```
looks like there's no magic checkbox to make tarball exports of the svn repos 
happen. I uploaded a .zip of r25 to the downloads section. I also gave you 
commit access to the svn repos in case you want to try that.

Mercurial is also an option instead of svn but I have no experience with that. 
Should be fine for a small project like this though, maybe an excuse to get my 
mercurial feet wet.
```

Original comment by `dere...@gmail.com` on 18 Aug 2010 at 9:35

**Comment by GoogleCodeExporter** [comment]
```
There is an svn client for hg that I can try to grok as well.  I'm sure we can 
work it out.  Thanks for the quick reply and looks like this issue can be 
closed.
```

Original comment by `slestak...@gmail.com` on 18 Aug 2010 at 9:51

**Comment by GoogleCodeExporter** [comment]
```
This is fixed in v0.2.4
```

Original comment by `dere...@gmail.com` on 11 Feb 2013 at 6:11
- Changed state: **Fixed**
